nxpprog.py

```diff
@@ -1683,7 +1542,145 @@ def signal_handler(sig, frame):
         prog.device.close()
     logging.error('Ctrl+C Pressed bailing out!')
     sys.exit(0)
+@click.command()
+@click.option(
+    '--binary',
+    required=True,
+    help='path to the firmware.bin file you want to program the board with.')
 
-if __name__ == '__main__':
+@click.option(
+    '--device',
+    required=True,
+    type=str,
+    default="",
+    help='Path to serial device file. In linux the name should be '
+    'something similar to "/dev/ttyUSB0", WSL "/dev/ttyS0", and '
+    'Max OSX "/dev/tty-usbserial-AJ20A5".')
+
+@click.option(
+    '--udp',
+    is_flag=True,
+    help='program processor using Ethernet.')
+
+@click.option(
+    '--cpu',
+    type=str,
+    default="",
+    help='set the cpu type.')
+
+@click.option(
+    '--osfreq',
+    type=int,
+    default=0,
+    help='set the oscillator frequency.')
+
+@click.option(
+    '--baud',
+    type=int,
+    default=0,
+    help='set the baud rate.')
+
+@click.option(
+    '--xonxoff',
+    is_flag=True,
+    help='enable xonxoff flow control.')
+
+@click.option(
+    '--control',
+    is_flag=True,
+    help='use RTS and DTR to control reset and int0.')
+
+@click.option(
+    '--start',
+    is_flag=True,
+    help='start the device at a set address.')
```

Comment:
Can you capitalize or lower case all of the help menu strings. Below we have "Enable" and here we have "start"

Reply:
I think I'd prefer them capitalized, but I'll let you decide.